service/dap/server.go

```diff
@@ -1651,24 +1654,20 @@ func (s *Server) getTypeIfSupported(v *proc.Variable) string {
 // variables request can be issued to get the elements of the compound variable. As a
 // custom, a zero reference, reminiscent of a zero pointer, is used to indicate that
 // a scalar variable cannot be "dereferenced" to get its elements (as there are none).
-func (s *Server) convertVariableEvaluate(v *proc.Variable, qualifiedNameOrExpr string) (value string, variablesReference int) {
-	return s.convertVariableWithOpts(v, qualifiedNameOrExpr, false, true)
-}
-
 func (s *Server) convertVariable(v *proc.Variable, qualifiedNameOrExpr string) (value string, variablesReference int) {
 	// If the client supports paging, we expect them to request the variables in reasonable chunks, so we do
 	// not want to display the loaded information.
-	return s.convertVariableWithOpts(v, qualifiedNameOrExpr, false, !s.clientCapabilities.supportsVariablePaging)
+	return s.convertVariableWithOpts(v, qualifiedNameOrExpr, false)
```

Comment:
You no longer need the comment above this.

Reply:
Done.